apps/howto_deploy/tvm_runtime_pack.cc

```diff
@@ -37,7 +37,9 @@
  *  You need to remember to change it to point to the right file.
  *
  */
+#define TVM_USE_LIBBACKTRACE 0
```

Comment:
Is it necessary to disable libbacktrace?

Reply:
yes, because we aren't linking to libbacktrace.

Reply:
That makes sense to me, thanks.